server/grpc/api/service.go

```diff
@@ -100,3 +103,101 @@ func fromProtoDependencies(deps []*definition.Dependency) []*service.Dependency
 	}
 	return ds
 }
+
+// ToProtoService converts an internal service struct to the protobuf definition
+// TODO: should not be public. Need to move server/grpc/service.go to server/grpc/api/service.go
+func ToProtoService(s *service.Service) *definition.Service {
```

Comment:
This function is already defined in `server/grpc/core/proto.go`, 

make it public or create a function in definition package.

Reply:
Good catch! I searched for those function but I didn't find them...
I removed to "old" ones in favor of these one because they are in the right place. The `server/grpc/core` will be deleted in the future.